### PR TITLE
Updated home page and added minor CSS styling for school navigation

### PR DESCRIPTION
**Pull request by philmillr**
Home page now includes additional sections (Screenshots below): 
- Title Section (existed before, but have been split out into an independent function) 
- Mission Statement (pure textbook with TEN mission statement - has been updated to most recent TEN mission statement)
- Problem Statement (taken nearly identically from Product 1. Refactored the pie chart with the help of ChatGPT) 
- Solution (Followed similar format to product 1, but includes updated language)
- Ending Logo - I think it would be great if we include that on most pages to give a consistent feel across the product

The public navigation page has minor CSS styling now. It will likely need to be updated by a designer with a better eye than me though haha. 

Mission: 
image

Problem Statement: 
image

Solution: (note it is longer than the screenshot below): 
image

Ending Logo:
image

CSS Styling for School Navigation Page: 
image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import ListingPage from './pages/Sell/ListingPage';
 import ConfirmationPage from './pages/Sell/ConfirmationPage';
 import { SellProvider } from './contexts/SellContext';
 import NavigationBar from './components/ExternalNavigationBar/ExternalNavBar';
```
Comment (HongjieWang98) [inline src/App.jsx]: It seems a bit confusing of the file paths for NavigationBar and the ExternalNavigationBar. Could you look into it, and potentially modify the path name for one of them, for future reference?

**Review by HongjieWang98 — CHANGES_REQUESTED** [review]
There is an error on the App.js file. Identifier 'CreateAccount' has already been declared twice.

File: src/App.jsx (hunk 1 above)
Comment (philmillr) [reply src/App.jsx]: sorry, what is confusing about the file path? Would it be clearer if the folder name was the same as the file?

Comment (philmillr) [reply src/App.jsx]: Sorry, and where did you see the 'CreateAccount' declared twice? I actually did not indent to edit the App.js file at all in this PR, so I wonder if it's a merge conflict that I didn't resolve correctly.

Comment (HongjieWang98) [reply src/App.jsx]: > sorry, what is confusing about the file path? Would it be clearer if the folder name was the same as the file?

These two below: Are they both ExternalNavBar? I know Stephen previously adapted an external navigation bar for internal use, which is why the names are quite similar.

import NavigationBar from './components/ExternalNavigationBar/ExternalNavBar';
import ExternalNavigationBar from './components/NavigationBar/ExternalNavBar';

Comment (HongjieWang98) [reply src/App.jsx]: > Sorry, and where did you see the 'CreateAccount' declared twice? I actually did not indent to edit the App.js file at all in this PR, so I wonder if it's a merge conflict that I didn't resolve correctly.

Line 13 and line 17 on the updated file. Hmm, it seems to be a merge conflict. 

image

Comment (philmillr) [reply src/App.jsx]: Oh this was definitely a merge conflict I didn't resolve properly haha. I do not see either of those on my local copy! Easy fix though!

**Review by HongjieWang98 — APPROVED** [review]

**Pull request merged**